src/runtime/tests/particle-api-test.ts

```diff
@@ -23,6 +23,7 @@ import {Runtime} from '../runtime.js';
 import {SingletonStore} from '../store.js';
 import {Speculator} from '../../planning/speculator.js';
 import {SingletonStorageProvider, CollectionStorageProvider, BigCollectionStorageProvider} from '../storage/storage-provider-base.js';
+import {collectionHandleForTest} from '../testing/handle-for-test.js';
 
 async function loadFilesIntoNewArc(fileMap: {[index:string]: string, manifest: string}): Promise<Arc> {
```

Comment:
do these tests work with the flag set to true? or not yet?

Reply:
Not yet (I didn't actually try it). There's still a lot of other code that needs to be updated before this will all work with the flag on.

Reply:
I'll clean up the additional inspector call when I come back to update the rest of this file (I think eventually the inspector will have to go, I'm not sure it's compatible with new-style stores)